lib/models/user.js

```diff
@@ -30,7 +30,12 @@ limitations under the License.
  * @prop {string} displayName The 'displayname' of the user if known.
  * @prop {string} avatarUrl The 'avatar_url' of the user if known.
  * @prop {string} presence The presence enum if known.
- * @prop {Number} lastActiveAgo The last time the user performed some action in ms.
+ * @prop {Number} lastActiveAgo The time elapsed in ms since the user interacted
+ *                proactively with the server, or we saw a message from the user
+ * @prop {Number} lastPresenceTs Timestamp (ms since the epoch) for when we last
+ *                received presence data for this user.  We can subtract
+ *                lastActiveAgo from this to approximate an absolute value for
+ *                when a user was last active.
```

Comment:
It would be useful to have a method on `User` to do this calculation rather than just mentioning it in the docs.